### PR TITLE
Shady nagy/net5

### PR DESCRIPTION
**Pull request by ShadyNagy**
Test Date added in Blazor.

### CODE REVIEW

File: FrontDesk/src/BlazorShared/Models/Room/RoomDto.cs [hunk 1]
```diff
@@ -6,5 +6,6 @@ public class RoomDto
     {
         public int RoomId { get; set; }
         public string Name { get; set; }
+        public string Color { get; set; }
```
Comment (ardalis) [inline FrontDesk/src/BlazorShared/Models/Room/RoomDto.cs]: Not sure this needs a color property. If you have a class in CSS like room1, room2, room3 then you could style each room just in the CSS and not need it in your shared model.

File: FrontDesk/src/FrontDesk.Api/Endpoints/Configuration/Read.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+namespace FrontDesk.Api.ConfigurationEndpoints
+{
+    public class Read : BaseAsyncEndpoint<string, string>
+    {
+        public Read()
+        {
+        }
+
+        [HttpGet("api/configurations/{configurationName}")]
```
Comment (ardalis) [inline FrontDesk/src/FrontDesk.Api/Endpoints/Configuration/Read.cs:15]: We don't have that much configuration - wouldn't it make sense to have the endpoint return all config values needed by the client app?

File: FrontDesk/src/FrontDesk.Core/Aggregates/Room.cs [hunk 3]
```diff
@@ -6,10 +6,12 @@ namespace FrontDesk.Core.Aggregates
     public class Room : BaseEntity<int>, IAggregateRoot
     {
         public virtual string Name { get; private set; }
+        public string Color { get; private set; }
 
-        public Room(string name)
+        public Room(string name, string color)
```
Comment (ardalis) [inline FrontDesk/src/FrontDesk.Core/Aggregates/Room.cs]: The core domain model doesn't need to know what the UI uses for the color of the room. It's not literally the room's color in real life.

**Review by ardalis — CHANGES_REQUESTED** [review]
Can we keep color as a UI-only concern, not a model and API concern?

**Review by ardalis — APPROVED** [review]

File: FrontDesk/src/FrontDesk.Api/Endpoints/Configuration/Read.cs (hunk 2 above)
Comment (ShadyNagy) [reply FrontDesk/src/FrontDesk.Api/Endpoints/Configuration/Read.cs:15]: Done

File: FrontDesk/src/FrontDesk.Core/Aggregates/Room.cs (hunk 3 above)
Comment (ShadyNagy) [reply FrontDesk/src/FrontDesk.Core/Aggregates/Room.cs]: Done

**Pull request merged**